File: task.clj
;; Check your code compilation here: https://www.jdoodle.com/execute-clojure-online/

(ns clojure.ia0x_hello
	(:gen-class))

(def firstName "Viktroiia")
(def lastName "Yakymenko")
(def groupName "IA-04")
(def greetings "Hello world")
(def orderNumber 25)

(def hello-world 
    (println (format "%s,%s %s and you are %d in %s", greetings, firstName, lastName, orderNumber, groupName)))

(hello-world)

File: task.pas
{ Check your code compilation result at: https://www.onlinegdb.com/online_pascal_compiler }

program Hello;

uses sysutils;

var 
    FirstName, LastName, Greetings: string;
    GroupNumber, OrderNumber: integer;
    Output: string;

begin
  FirstName := 'Rostyslav';
  LastName := 'Nazarenko';
  GroupNumber := 1;
  OrderNumber := 18;
  Greetings := 'Hello world';
  
  Output := Greetings + ', ' + FirstName + ' ' + LastName + ', IA-0' + IntToStr(GroupNumber) + ', Order: ' + IntToStr(OrderNumber);  
  
  writeln(Output)
end.